perf/get.go

```diff
@@ -25,7 +25,7 @@ type GetOptions struct {
 	Execution *int
```

Comment:
Add a field `Count bool` so we know this explicitly getting the count of perf tests.